### PR TITLE
Add label of a group name to rule_group_iterations_missed_total metric 

### PR DESCRIPTION
**Pull request by homelessnessbo**
Signed-off-by: Ivan Kiselev <kiselev_ivan@pm.me>

Right now it's impossible to understand what group of rules is struggling without manually analyzing config. Adding `rule_group ` should make this process a bit easier.

### CODE REVIEW

File: rules/manager.go [hunk 1]
```diff
@@ -102,16 +102,19 @@ func NewGroupMetrics(reg prometheus.Registerer) *Metrics {
 			Help:       "The duration of rule group evaluations.",
 			Objectives: map[float64]float64{0.01: 0.001, 0.05: 0.005, 0.5: 0.05, 0.90: 0.01, 0.99: 0.001},
 		}),
-		iterationsMissed: prometheus.NewCounter(prometheus.CounterOpts{
-			Namespace: namespace,
-			Name:      "rule_group_iterations_missed_total",
-			Help:      "The total number of rule group evaluations missed due to slow rule group evaluation.",
-		}),
 		iterationsScheduled: prometheus.NewCounter(prometheus.CounterOpts{
```
Comment (bwplotka) [inline rules/manager.go]: Can we change this one as well?

**Comment by brian-brazil** [comment]
This counts as a breaking change, you can already figure this out from rule_group_last_duration_seconds.

**Comment by homelessnessbo** [comment]
@brian-brazil then, to come up with an actual alert - another metric is needed that actually reflects the evaluation interval for certain group

**Comment by brian-brazil** [comment]
rule_group_interval_seconds covers that.

**Comment by homelessnessbo** [comment]
yep. You're totally right, the combination of those two gives the opportunity to alert. 

Thanks!

**Comment by bwplotka** [comment]
Wow, I just noticed it's not a Thanos project 0.o I think this is a good sign ;p

**Pull request closed without merging**